lib/cli.js

```diff
@@ -423,7 +433,7 @@ internals.options = function () {
     options.paths = argv._ ? [].concat(argv._) : options.paths;
 
     const keys = ['assert', 'bail', 'colors', 'context-timeout', 'coverage', 'coverage-exclude',
-        'coverage-path', 'default-plan-threshold', 'dry', 'environment', 'flat', 'globals', 'grep',
+        'coverage-path', 'coverage-all', 'coverage-flat', 'default-plan-threshold', 'dry', 'environment', 'flat', 'globals', 'grep',
```

Comment:
Somewhere later in the `internals.options` function can you also add a check for if `coverage-flat` and not `coverage-all` then log an error and exit. Since the `coverage-flat` requires the `coverage-all` flag I want to make sure we catch the case when its not provided. Or do you think we should set `--coverage-all` if `--coverage-flat` is provided?

Reply:
I think the error and exit would be the correct way to go, it should be explicit with what's going on :+1:

Reply:
I have also added to the options to enable coverage when `--coverage-all` is set, in a similar behaviour to `--threshold` and some of the reporters